### ISSUE TITLE
Exercises have a submit button input control that is focusable WELL outside the viewport but has aria-hidden=true

### ISSUE THREAD

**Issue by bedmison** [OpenDSA/OpenDSA#616]
This is specifically related to KA exercises. The actual submit button has this HTML:

`<input style="position: absolute; left: -9999px; width: 1px; height: 1px" type="submit" aria-hidden="true">`

which puts the button offscreen left, and the "submit" button that is visible manipulates the real submit via JS to cause it to send the request. Even though the button is off screen, it is still in the DOM and still focusable. Not real sure what the design decision was behind this, but this issue is in every book/canvas page that has a KA exercise. 

Fix: 
Ensure aria-hidden elements are not focusable nor contain focusable elements. 

To solve this problem, you need to fix the following:
- Focusable content should be disabled or be removed from the DOM

**Comment by bedmison** [comment]
Assigned to Miheer.
